src/gt4py/cartesian/gtc/dace/utils.py

```diff
@@ -26,6 +26,14 @@
 from gt4py.cartesian.gtc.passes.oir_optimizations.utils import compute_horizontal_block_extents
 
 
+def array_strides(fld: str, dims: List[str]):
```

Comment:
I know those functions (`array_strides` and `array_total_size`) where lifted from the original code, but they seem very brittle, considering we are using the internal of the dace symbolic parser. 

Can we compute this more cleanly and/or use an existing function within DaCe?

Reply:
I didn't see the parse strides function in DaCe internal, maybe we could keep the original in the code?